zipkin-lens/src/zipkin/trace.js

```diff
@@ -291,7 +291,12 @@ function addLayoutDetails(
   }
 }
 
-export function detailedTraceSummary(root, logsUrl) {
+export function detailedTraceSummary(
+  root,
+  logsUrl,
+  archivePostUrl,
+  archiveUrl,
+) {
```

Comment:
Are these change still needed?

Reply:
I think so, as we still need to pass along the config values.

Reply:
All configurations seems possible to be retrieved from `useConfig` hooks.
like: https://github.com/openzipkin/zipkin/pull/3018/files#diff-bb9481d8056808f1f23f6f80ae8a10f6R102

And these parameters(`archivePostUrl` and `archiveUrl`) does not seem to be used anywhere...

Reply:
should I delete also logsUrl then?

Reply:
maybe...

Reply:
Nice detective work finding this file - for reference in #2985 I didn't touch this file. I guess it's copy-pasted from somewhere and indeed these URL variables aren't used, so let's go ahead and delete